RFS/src/main/java/org/opensearch/migrations/bulkload/lucene/LuceneDocumentsReader7.java

```diff
@@ -1,377 +0,0 @@
-package org.opensearch.migrations.bulkload.lucene;
-
-import java.io.IOException;
-import java.nio.file.Path;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Comparator;
-import java.util.List;
-import java.util.concurrent.atomic.AtomicReference;
-import java.util.function.Function;
-import java.util.function.Supplier;
-import java.util.stream.Collectors;
-
-import org.opensearch.migrations.bulkload.common.RfsLuceneDocument;
-import org.opensearch.migrations.bulkload.common.Uid;
-
-import lombok.Lombok;
-import lombok.RequiredArgsConstructor;
-import lombok.Value;
-import lombok.extern.slf4j.Slf4j;
-import org.reactivestreams.Publisher;
-import reactor.core.publisher.Flux;
-import reactor.core.publisher.Mono;
-import reactor.core.scheduler.Scheduler;
-import reactor.core.scheduler.Schedulers;
-import shadow.lucene7.org.apache.lucene.document.Document;
-import shadow.lucene7.org.apache.lucene.index.DirectoryReader;
-import shadow.lucene7.org.apache.lucene.index.IndexReader;
-import shadow.lucene7.org.apache.lucene.index.LeafReader;
-import shadow.lucene7.org.apache.lucene.index.LeafReaderContext;
-import shadow.lucene7.org.apache.lucene.index.SegmentCommitInfo;
-import shadow.lucene7.org.apache.lucene.index.SegmentReader;
-import shadow.lucene7.org.apache.lucene.index.SoftDeletesDirectoryReaderWrapper;
-import shadow.lucene7.org.apache.lucene.store.FSDirectory;
-import shadow.lucene7.org.apache.lucene.util.BytesRef;
-
-@RequiredArgsConstructor
-@Slf4j
-public class LuceneDocumentsReader7 implements LuceneDocumentsReader {
```

Comment:
Github is showing that this file was deleted, but it's effectively a rename to LuceneIndexReader, right?

Reply:
Yup